src/libsnark_helpers/libsnark_helpers.hpp

```diff
@@ -20,47 +20,32 @@
 
 #include <libff/common/default_types/ec_pp.hpp>
 
-// Contains required interfaces and types (keypair, proof, generator, prover, verifier)
-#include <libsnark/zk_proof_systems/ppzksnark/r1cs_ppzksnark/r1cs_ppzksnark.hpp>
 #include <libsnark/gadgetlib1/gadget.hpp>
+// Contains required interfaces and types (keypair, proof, generator, prover, verifier)
+#include "zeth.h"
+#include "snarks_alias.hpp"
+#include "snarks_import.hpp"
 
-namespace libzeth {
-
-// -- Defined in the CPP file -- //
-libff::bigint<libff::alt_bn128_r_limbs> libsnarkBigintFromBytes(const uint8_t* _x);
-std::string HexStringFromLibsnarkBigint(libff::bigint<libff::alt_bn128_r_limbs> _x);
-std::string outputPointG1AffineAsHex(libff::alt_bn128_G1 _p);
-std::string outputPointG2AffineAsHex(libff::alt_bn128_G2 _p);
-
-boost::filesystem::path getPathToSetupDir();
-boost::filesystem::path getPathToDebugDir();
+#include "extended_proof.hpp"
 
-bool replace(std::string& str, const std::string& from, const std::string& to);
+namespace libzeth {
 
-// -- Defined in the TCC file -- //
+//old
```

Comment:
Is the "old" comment saying that the functions below should be deleted?

Reply:
Just a comment I put there to differentiate old from new function I temporarily added.